src/Components/BadgesAndIcons.jsx

```diff
@@ -0,0 +1,31 @@
+import PropTypes from "prop-types";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faPencilAlt, faTrash } from "@fortawesome/free-solid-svg-icons";
+
+function BadgesAndIcons(props) {
```

Comment:
you cannot write the  whole functionality in common components
you need to create the separate component for icon and badge 
you have merged this

Reply:
ma'am earlier i made separate file but you that no need of this file that's why i remove that file